src/components/studies/participants/ParticipantManager.tsx

```diff
@@ -856,7 +897,16 @@ const ParticipantManager: FunctionComponent<ParticipantManagerProps> = () => {
             </CollapsibleLayout>
           </Box>
         </Box>
-
+        <BatchEditForm
+          isEnrolledById={false}
```

Comment:
while it is true that it will be false since this will only show when they are enrolled by phone, it would be more consistent to check using isSignInById(study.signInTypes)